### DISCUSSION TITLE
Aptos sent to sui wallet

### DISCUSSION THREAD

**Discussion by ccarnegie41** [aptos-labs/aptos-developer-discussions#526] [Questions] [answered]
### Describe your question in detail.

I sent 5 aptos to sui wallet unknowingly. It did not show up in sui wallet but was sent succesfully and when transaction viewed on intermet explorer upon opening the receiver they are there under aptos mainnet. Am i somehow able to get them back? Sent from kracken if that makes any difference
![Screenshot_20241113_182702_Chrome](https://github.com/user-attachments/assets/9e70edb4-449e-4280-b8af-34e25717dad8)
![Screenshot_20241113_091314_Chrome](https://github.com/user-attachments/assets/5f6b39e4-1c56-433f-9267-8f6cccb0345f)

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by 0xbrw** [comment, marked as answer]
Unfortunately there is nothing we can do here since these are separate blockchains that are not compatible. Next time be careful when you transfer funds between wallets/networks.

**Comment by ccarnegie41** [comment]
Ok ty and yes learned my lesson. Still shows as being in that wallet but i
guess im fubared unless sui wallet opens up to other chains?

Regardless i appreciates the response

Craig Carnegie
